### DISCUSSION TITLE
Adding new slide to Fancybox via Ajax

### DISCUSSION THREAD

**Discussion by asmith20002** [fancyapps/ui#146] [Q&A]
Hello,

The question is mainly about adding new slide on the fly.  A fancybox is created with one slide but more slides should be added by events. I can't find this in the fancybox doc.

Let's say I have my own html:
```
		Fancybox.show([{
			src: '#my_custom_html_content',
			type: "inline",
		      },
		]});
```

The html have a link/button to load new data via ajax. Based on the ajax success data I want to add/create a new html slide to the currently visible Fancybox and then next() to the newly created slide.

How do I add the new slide?

**Comment by fancyapps** [comment]
Hi,

There are currently no API methods for that, but it is planned to implement them in the upcoming releases.
